style.css

```diff
@@ -0,0 +1,100 @@
+#id-main-content {
+  box-shadow: 0 0 5px black;
+  width: 300px;
+  height: auto;
+  margin: auto;
+  background-image: url(/images/image_geometry_1.png);
+  background-size: 90px;
+  background-repeat: no-repeat;
+  background-attachment: initial;
+  display: flex;
+  flex-direction: column;
+}
+
+#id-header-content {
+  box-shadow: 0 0 5px black;
```

Comment:
- [ ] [OPTIONAL] The original design doesn't have a box shadow, please remove it